boards/shared/Src/Io/Io_SharedErrorTable.c

```diff
@@ -71,38 +71,37 @@ static void Io_ProcessFsmNonCriticalErrorMsg(
 {
     SET_ERROR(
         error_table, FSM_NON_CRITICAL_PAPPS_OUT_OF_RANGE,
-        data->papps_out_of_range == 0);
```

Comment:
All the `== 0` are wrong. Not sure how I didn't notice that before ... Oh  well that's why we have tests.